### PR TITLE
Adding Terraform Sample Code for deploying Druid, and Presto on GCP using DataProc

### PR DESCRIPTION
**Pull request by pupamanyu**

### CODE REVIEW

File: examples/terraform/druid-migration/README.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# Launch Druid on Compute Engine using Terraform
+
+Using this code repository users can launch there  Druid cluster on GCP using terraform.
```
Comment (morgante) [inline examples/terraform/druid-migration/README.md]: Spelling/grammar.

File: examples/terraform/druid-migration/README.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    * roles/storage.admin
+    
+    * roles/cloudsql.admin
+ 
+    * roles/dataproc.editor
+    
+	
+## Input variables
+
+Below are the variables which we are using while deploying the infrastructure using terraform.
```
Comment (morgante) [inline examples/terraform/druid-migration/README.md]: I suggest autogenerating this using terraform-docs.

File: examples/terraform/druid-migration/terraform.tfstate [hunk 3]
```diff
@@ -0,0 +1,2096 @@
+{
```
Comment (morgante) [inline examples/terraform/druid-migration/terraform.tfstate]: Never commit tfstate.

File: examples/terraform/druid-migration/README.md (hunk 2 above)
Comment (pupamanyu) [reply examples/terraform/druid-migration/README.md]: Generated and committed changes on my forked repo

File: examples/terraform/druid-migration/terraform.tfstate (hunk 3 above)
Comment (pupamanyu) [reply examples/terraform/druid-migration/terraform.tfstate]: Removed .tfstate* files

**Comment by pupamanyu** [comment]
Can we please complete the review. This is critical pull request for two documentation assets

File: examples/terraform/druid-migration/variables.tf [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+  default     = "<kafka_node_name>"
+}
+
+variable kafka_disk {
+  description = "Kafka Operating System."
+  default     = "ubuntu-os-cloud/ubuntu-1604-lts"
+}
+
+variable zookeeper_ip_list {
+  description = "This variable is used internally for storing the list of Zookeeper clusters IP Address"
```
Comment (morgante) [inline examples/terraform/druid-migration/variables.tf]: If a variable is only used internally, why are we exposing it? Variables should only be used for items which users are expected or able to override/edit.

File: examples/terraform/druid-migration/variables.tf [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+  default     = "n1-standard-32"
+}
+
+variable dataproc_worker_boot_disk_size_gb {
+  description = "Size of the disk for dataproc worker node."
+  default     = 500
+}
+
+variable mod_startup_script {
+  description = "This variable is used internally."
```
Comment (morgante) [inline examples/terraform/druid-migration/variables.tf]: No "internal" variables please.

File: examples/terraform/druid-migration/zookeeper/main.tf [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  }
+
+  network_interface {
+    network       = "${var.network}"
+    access_config = {}
+  }
+
+  metadata {
+    startup-script = "echo ${count.index + 1} > /usr/share/zookeeper.id; gsutil cp gs://${var.bucket_name}/zookeeper_script.sh /opt; chmod +x /opt/zookeeper_script.sh; /opt/./zookeeper_script.sh"
```
Comment (morgante) [inline examples/terraform/druid-migration/zookeeper/main.tf]: Startup scripts can natively be pulled from GCS: https://cloud.google.com/compute/docs/startupscript#cloud-storage

**Comment by morgante** [comment]
@pupamanyu Ping - did you see the review feedback above?

**Pull request closed without merging**